### PR TITLE
[settings_improvement][ADD] some features to help workflows blocking det...

### PR DESCRIPTION
**Pull request by Olivier-LAURENT**
...ection and repairing

### CODE REVIEW

File: settings_improvement/workflow_view.xml [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+                    <field name="id"/>
+                    <group expand="0" string="Group By...">
+                        <filter name="state" string="State" context="{'group_by': 'state'}" />
+                    </group>
+                </xpath>
+            </field>
+        </record>
+
+        <record id="view_workflow_instance_tree" model="ir.ui.view">
+            <field name="name">workflow.instance.tree (Seetings Improvement)</field>
```
Comment (laetitia-gangloff) [inline settings_improvement/workflow_view.xml]: replace (Seetings Improvement) by (settings_improvement)

File: settings_improvement/workflow_view.xml [hunk 2]
```diff
@@ -0,0 +1,150 @@
...
+            <field name="inherit_id" ref="base.view_workflow_instance_tree"/>
+            <field name="arch" type="xml">
+                <xpath expr="//tree" position="inside">
+                    <field name="id"/>
+                </xpath>
+            </field>
+        </record>
+
+        <record id="view_workflow_workitem_tree" model="ir.ui.view">
+            <field name="name">workflow.workitem.tree (Seetings Improvement)</field>
```
Comment (laetitia-gangloff) [inline settings_improvement/workflow_view.xml]: replace (Seetings Improvement) by (settings_improvement)

**Comment by laetitia-gangloff** [comment]
:+1:

**Comment by anthony-muschang** [comment]
:+1:

**Pull request merged**